.vscode/launch.json

```diff
@@ -5,17 +5,19 @@
 			"type": "chrome",
 			"request": "launch",
 			"name": "Playground: Browser",
-			"url": "http://localhost:10001"
+			"url": "http://localhost:10001",
+			"webRoot": "${workspaceFolder}"
```

Comment:
I don't exactly know what this option is used for but from its description it sounds like it should point to the directory that the files are served from and pointing it to the workspace folder would be incorrect. Could you please explain what this does?